src/main/scala/com/sky/kafka/configurator/Main.scala

```diff
@@ -42,22 +43,26 @@ object Main extends LazyLogging {
     parse(args).flatMap { conf =>
       val configurator = TopicConfigurator.reader(conf)
       val result = configureTopicsFromFile(conf.file, configurator)
-
+        .recoverWith {
+          case NonFatal(throwable) =>
+            logger.error("Error whilst configuring topics", throwable)
```

Comment:
Shouldn't it be a warning as we can recover from it?

Reply:
Actually it might make sense as it's in the main, and it's the only operation that the program does.